source/code-generation/query-converter/convert-stored-procedures.txt

```diff
@@ -10,3 +10,277 @@ Convert Stored Procedures
    :depth: 1
    :class: singlecol
 
+You can import and convert your SQL stored procedures to MongoDB syntax 
+with the query converter. The query converter considers the mapping 
+rules and schema transformations defined in your project when converting
+your SQL code.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+Before you Begin
+----------------
+
+.. include:: /includes/fact-query-converter-disclaimer.rst
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Navigate to the query converter pane
+
+  	   From the :guilabel:`Code Generation` tab, click the :guilabel:`Query Converter` pane.
+
+   .. step:: Open the query converter view
+
+      -  If it is your first time using the query converter in your
+         project, click :guilabel:`Import From Database`.
+
+      -  If your project already has converted SQL code, click the
+         :guilabel:`Manage Database Objects` button on the left-hand
+         pane.
+
+   .. step:: Connect to your relational database
+
+      a. Specify the connection details to your source database. 
+         
+         For details, see :ref:`rm-relational-database-connection-strings`.
+      #. Click :guilabel:`Connect`.
+
+   .. step:: Select stored procedures
+
+      a. On the :guilabel:`Import Database Objects` modal, click
+         the :icon-fa5:`chevron-right` icon next to :guilabel:`Database`.
+
+      #. Click the :icon-fa5:`chevron-right` icon next to your schema.
+
+      #. Click the :icon-fa5:`chevron-right` icon next to
+         :guilabel:`Stored Procedures`.
+
+         .. tip::
+
+            You can toggle stored procedures for conversion by clicking
+            the :icon-fa5:`check-square` icon next to a stored procedure's
+            name. All stored procedures are selected by default.
+
+      #. Click :guilabel:`Save`.
+
+         The SQL syntax for each stored procedure in your database 
+         schema is imported into your project and is visible in the 
+         :guilabel:`Query Converter` pane under 
+         :guilabel:`Stored Procedures`.
+
+   .. step:: Convert and test syntax
+
+      a. Click a stored procedure's name from the left-hand pane under
+         :guilabel:`Stored Procedures`.
+         
+         The SQL syntax displays in the 
+         :guilabel:`Imported Stored Procedure` pane.
+
+      #. Select a :guilabel:`Target Language` from the drop-down.
+         
+         Query converter supports stored procedure conversions to
```

Comment:
This warrants a longer explanation, probably outside of the procedure. MongoDB doesn't have stored procedures per se, so users need to figure out where they want to host this logic once converted. The main options are to host in their app's backend, or they could use App Services Functions. Users should choose the programming language that matches the environment where the code will be hosted. The converted code will use the selected language and have MongoDB queries/aggregations interleaved.

Reply:
[I added a version of your blurb to the ](https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-32086/code-generation/query-converter/convert-stored-procedures/#before-you-begin) `Before you Begin` section outlining these major points and pointing the the app function docs, LMKYWT.